staging/src/k8s.io/legacy-cloud-providers/azure/azure_standard.go

```diff
@@ -29,6 +29,8 @@ import (
 	"strconv"
 	"strings"
 
+	"k8s.io/klog"
```

Comment:
This line prevents from building the image, `klog` is imported just a few lines below